ndc-reference/bin/reference/main.rs

```diff
@@ -100,15 +109,17 @@ async fn metrics_middleware<T>(
 // ANCHOR_END: metrics_middleware
 // ANCHOR: init_app_state
 fn init_app_state() -> AppState {
-    // Read the CSV data files
-    let articles = read_csv("articles.csv").unwrap();
-    let authors = read_csv("authors.csv").unwrap();
+    // Read the JSON data files
+    let articles = read_json_lines("articles.json").unwrap();
+    let authors = read_json_lines("authors.json").unwrap();
+    let universities = read_json_lines("universities.json").unwrap();
```

Comment:
Complete and utter nitpick (sorry) but we could consider renaming this to `institutions` to match things like BibTeX, and because not every author is associated with a university.